apps/web/components/eventtype/EventLimitsTab.tsx

```diff
@@ -198,96 +196,6 @@ function RollingLimitRadioItem({
   );
 }
 
-const MinimumBookingNoticeInput = React.forwardRef<
-  HTMLInputElement,
-  Omit<UseFormRegisterReturn<"minimumBookingNotice">, "ref">
```

Comment:
Move MinimumBookingNotice into own file and reuse it for seatsMinimumBookingNotice